tests/ProxyManagerTest/Functional/MultipleProxyGenerationTest.php

```diff
@@ -119,6 +121,8 @@ public function getTestedClasses() : array
         return [
             [BaseClass::class],
             [ClassWithMagicMethods::class],
+            [ClassWithHintedMagicMethods::class],
+            //[InterfaceWithHintedMagicMethods::class],
```

Comment:
This test doesn't verify interface proxying - need to move to appropriate functional tests instead.